Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -1408,6 +1408,7 @@ namespace AzToolsFramework
             {
                 const AZ::EntityId entityId = entity->GetId();
                 const AZStd::string nestedEntityAliasPath = m_instanceToTemplateInterface->GenerateEntityAliasPath(entityId);
+                detachedEntityAliasPaths.push_back(AZStd::move(nestedEntityAliasPath));
```

Comment:
nit: I think it's good to add this to the paths after the actual detaching happens. If this code changes in the future and we introduce an early return between this and the actual detach, it can cause issues.